libc/test/src/math/smoke/CMakeLists.txt

```diff
@@ -3655,6 +3655,54 @@ add_fp_unittest(
     libc.src.math.totalordermagf16
 )
 
+add_fp_unittest(
+  totalordermag_test
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    totalordermag_test.cpp
+  HDRS
+    TotalOrderMagTest.h
+  DEPENDS
+    libc.src.math.totalordermag
+)
```

Comment:
Nit: the order should be `totalordermag`, `totalordermagf`, `totalordermagl`, `totalordermagf16`, `totalordermagf128` here as well.